src/settings.hpp

```diff
@@ -28,6 +28,8 @@ class SettingsManager {
 #endif
         bool Unreads { true };
 
+        bool HideToTray { true };
```

Comment:
```suggestion
        bool HideToTray { false };
```
should be opt-in since not all environments will support it properly